include/unifex/when_all.hpp

```diff
@@ -255,13 +275,16 @@ struct _op<Receiver, Senders...>::type {
     }
   }
 
+  static constexpr std::size_t callback_running_bit{1};
   std::tuple<optional<value_variant_for_sender<remove_cvref_t<Senders>>>...> values_;
   optional<error_types<variant, remove_cvref_t<Senders>...>> error_;
-  std::atomic<std::size_t> refCount_{sizeof...(Senders)};
+  // count multiplied by to to allow add 1 when stop is requested
```

Comment:
Nit: "two", not "to"
```suggestion
  // count multiplied by two to allow add 1 when stop is requested
```
Alternatively,
```suggestion
  // count left-shifted by one to allow add 1 when stop is requested
```